docs/timelines/timeline-options.md

```diff
@@ -0,0 +1,104 @@
+---
+id: timeline-options
+title: Timeline Options
+sidebar_label: Timeline options
+---
+
+Timelines have a number of options which set how they operate. 
+Press \<Options\> (or the \[Options\] softkey on the top level menu) then the **select** button for the timeline you wish to edit.
+
+--- 
+
+## Handle Tab
+
+All options are the same as for a cue, see [Options](../cues/playback-options.md#handle-tab).
+
+--- 
+
+## Times Tab
```

Comment:
Maybe reference the context tool that can be used to crop the start/duration? #87423